### PR TITLE
add templates for CoC, Contributing and Issues

### PR DESCRIPTION
**Pull request by dannycolin**
Related to #8

### CODE REVIEW

**Review by alexmorley — CHANGES_REQUESTED** [review]
I think if we leave out the contributing.md (that'll take a little bit of work), then we can merge this so that the others are in place.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting any of the project team at (Alex: alex0morley@gmail.com, Danny: XX@X.com, ?). All
```
Comment (alexmorley) [inline CODE_OF_CONDUCT.md]: @johav Would you be OK to be on this list?

**Comment by dannycolin** [comment]
> I think if we leave out the contributing.md (that'll take a little bit of work), then we can merge this so that the others are in place.

Or we could add a note at the top. So if anyone has the time to do it before me then s/he'll have a template instead of starting from scratch. What do you thing?

**Comment by alexmorley** [comment]
> > I think if we leave out the contributing.md (that'll take a little bit of work), then we can merge this so that the others are in place.

> Or we could add a note at the top. So if anyone has the time to do it before me then s/he'll have a template instead of starting from scratch. What do you thing?

Sounds like a good idea.

**Comment by dannycolin** [comment]
> Sounds like a good idea.

Done. :wink:

**Review by alexmorley — APPROVED** [review]

**Pull request merged**